contracts/metagov/AngleDelegatorPCVDeposit.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "./SnapshotDelegatorPCVDeposit.sol";
+import "./utils/VoteEscrowTokenManager.sol";
+import "./utils/LiquidityGaugeManager.sol";
+import "./utils/OZGovernorVoter.sol";
+
+/// @title ANGLE Token PCV Deposit
+/// @author Fei Protocol
+contract AngleDelegatorPCVDeposit is SnapshotDelegatorPCVDeposit, VoteEscrowTokenManager, LiquidityGaugeManager, OZGovernorVoter {
```

Comment:
does this still need OZGovernorVoter?

Reply:
Discussed in private, we'll keep it just in case Angle restore some kind of on-chain governance in the next 4 years